src/plugins/layout/nodeinfo.ts

```diff
@@ -1,50 +1,22 @@
-import { ILayoutState, INodeInfo } from "@clarity-types/layout";
-import { createLayoutState, getNodeIndex, Tags } from "./stateprovider";
+import { INodeInfo } from "@clarity-types/layout";
+import { shouldIgnoreNode } from "./states/ignore";
+import { isCssText } from "./states/text";
 
-export const ForceMaskAttribute = "data-clarity-mask";
+export const UnmaskAttribute = "data-clarity-unmask";
 
 export function createNodeInfo(node: Node, parentInfo: INodeInfo): INodeInfo {
-  let index = getNodeIndex(node);
-  let ignore = shouldIgnoreNode(node, parentInfo ? parentInfo.ignore : false);
-  let forceMask = shouldForceMaskNode(node, parentInfo ? parentInfo.forceMask : false);
-  let state: ILayoutState = createLayoutState(node, ignore, forceMask);
-  return { index, ignore, forceMask, state };
+  let ignore = shouldIgnoreNode(node, parentInfo);
+  let unmask = shouldUnmaskNode(node, parentInfo);
+  let isCss = isCssText(node);
```

Comment:
>isCssText [](start = 14, length = 9)

Where did this go?

Reply:
isCssText function? It's in states/text.ts

---
In reply to: [260854116](https://github.com/Microsoft/clarity-js/pull/137#discussion_r260854116) [](ancestors = 260854116)